va/tlsalpn.go

```diff
@@ -185,7 +185,7 @@ func (va *ValidationAuthorityImpl) validateTLSALPN01(ctx context.Context, identi
 		return validationRecords, problem
 	}
 
-	if !cs.NegotiatedProtocolIsMutual || cs.NegotiatedProtocol != ACMETLS1Protocol {
+	if cs.NegotiatedProtocol != ACMETLS1Protocol {
```

Comment:
Reference for anyone checking this diff: https://pkg.go.dev/crypto/tls#ConnectionState

 	// NegotiatedProtocolIsMutual used to indicate a mutual NPN negotiation.
	//
	// Deprecated: this value is always true.
	NegotiatedProtocolIsMutual bool